### ISSUE TITLE
NTR: Lens: crystals in anterior surface (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7981]
New term request Lens: crystals in anterior surface (elthat_100520025220):   UMLS: 1859681

**Comment by pnrobinson** [comment]
@MickeySegal -- what is the disease context? I am searching for a definition

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1400/
"Late ocular complications. Crystal deposition in the anterior chamber, iris, ciliary body, choroid, fundus, and optic nerve manifests as Anterior segment problems. Crystals in the anterior lens surface, band keratopathy, peripheral corneal neovascularization, and posterior synechiae;"

**Comment by pnrobinson** [comment]
@MickeySegal 
It seems there are so many different locations where crystals can appear in Cystinosis, I am not sure it is good to have a separate term for each of them?

**Comment by MickeySegal** [comment]
It would be good to have a finding for the eye, since an ophthalmologist can observe this.  But I agree that the term used was too specific.  How about "Eye: crystal deposition" with an elaboration to include lens, iris, ciliary body, choroid, fundus and optic nerve?

We have analogous crystal terms at the level of the kidney, joint, small intestine, liver and bone marrow.  

We have a NTR request pending also for "Conjunctival biopsy: cystine crystals"; it might make sense to consolidate.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000955
- label: Ocular cystine crystal deposition
- parents: [Abnormal eye morphology (HP:0012372)](https://hpo.jax.org/app/browse/term/HP:0012372)
- pmid: PMID:30242621
- def: Accumulation of cystine crsytals in the eye. Patients present with crystal accumulation affecting the cornea and conjunctiva, but crystals can also be deposited in the iris, ciliary body, choroid and retinal pigment epithelium (RPE), and lens capsule. Corneal crystal deposition starts in infancy in the anterior periphery of the cornea, and progresses centrally and posteriorly. The crystals, which have a fine, uniform needle-shaped structure, are diffusely distributed within the corneal tissues.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7981
- synonyms:
